exercise2/assets/style.css

```diff
@@ -0,0 +1,47 @@
+.header{
+	font-size: 46px;
+	width: 100%;
+	height: auto;
+}
+.navigation {
+font-size: 10px;
+width:25%;
+float: left;
+}
+.contain{
+	width: 100%;
+	height: 80vh;
```

Comment:
i am used using vh for  relative height. If  use % for heigth it does not work. 
But if you think is better i can change it  for px. :)